ProductList/productList.html

```diff
@@ -0,0 +1,189 @@
+                    <label for="yellow">Yellow</label></div>
+                    <div class="radio-group">
+                        <input type="checkbox" class="radio" id="green" name="radio-btn" />
+                        <label for="green">Green</label></div>
+                    <div class="radio-group">
+                            <input type="checkbox" class="radio" id="blue" name="radio-btn" />
+                            <label for="blue">Blue</label></div>
+                    <div class="radio-group">
+                                <input type="checkbox" class="radio" id="indigo" name="radio-btn" />
+                                <label for="indigo">Indigo</label></div>
+                    <div class="radio-group">
+                                    <input type="checkbox" class="radio" id="violet" name="radio-btn" />
+                                    <label for="violet">Violet</label></div>
+                </div></div>
+                <div class="flex-container flex-columns pb"><h3 class="pb">Rating</h3>
+            <div class="radio-container  flex-container flex-columns pb mb">
+                <div class="radio-group">
+                    <input type="radio" class="radio" id="4+" name="radio-btn" />
+                    <label for="4+">4 stars and above</label></div>
+                    <div class="radio-group">
+                    <input type="radio" class="radio" id="3+" name="radio-btn" checked />
+                    <label for="3+">3 stars and above</label></div>
+                    <div class="radio-group">
+                    <input type="radio" class="radio" id="2+" name="radio-btn" />
+                    <label for="2+">2 stars and above</label></div>
+                    <div class="radio-group">
+                        <input type="radio" class="radio" id="1+" name="radio-btn" />
+                        <label for="1+">1 stars and above</label></div>
+                </div></div>
+                <div class="flex-container flex-columns pb"><h3 class="pb">Sort By</h3>
+            <div class="radio-container flex-container flex-columns pb mb">
+                <div class="radio-group">
+                    <input type="radio" class="radio" id="lth" name="radio-btn" />
+                    <label for="lth">Price-Low to High</label></div>
+                    <div class="radio-group">
+                    <input type="radio" class="radio" id="htl" name="radio-btn" checked />
+                    <label for="htl">Price-High to Low</label></div>
+                </div></div>
+        </nav>
+        <main class="ml mt flex-container flex-columns">
+            <h2 class="brand">Showing All products<span class="text-small">(Showing 7 products)</span></h2>
+            <div id="card-container" class="mt flex-columns">
+                <div class="card card-basic card-shadow">
+                    <img src="https://picsum.photos/id/1005/340/196" class="card-img-top" alt="...">
```

Comment:
Give some name to **alt** attribute.

Reply:
Thank you for your suggestion, i will add name to it